logproof/src/bfv_statement.rs

```diff
@@ -823,32 +831,62 @@ mod tests {
 
             // statements without duplicate messages
             for (i, m) in messages.iter().enumerate() {
-                let (ct, u, e, r) = self
+                if i < num_unique_public_statements {
+                    let (ct, components) = self
+                        .encryptor
+                        .encrypt_return_components(&m.plaintext)
```

Comment:
I guess I'm not opposed to that for this particular method, which exists on the `seal_fhe::Encryptor` and is not part of the public API. However I wouldn't want to change the names of the `pub` methods on the runtime, because for most users it is simpler to not really consider asym vs sym and just default to asym.

Reply:
Hmm that's fair. The asymmetry (har har) is a tad odd but I agree that it doesn't make sense to change the public names.